ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -27,6 +27,7 @@ <h3 class="title">OBJECT VIEWER</h3>
     </div>
   </div>
 </div>
```

Comment:
In UI after adding index with 1 column and checking the index details, its showing all the columns in it

Reply:
Will this be addressed as part of backlog?

Reply:
This issue is fixed. It works fine with the current code.